ChemistryLib/PhreeqcIOData/ChemicalSystem.h

```diff
@@ -0,0 +1,45 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <memory>
+#include <vector>
+
+namespace MeshLib
+{
+class Mesh;
+}
+
+namespace ChemistryLib
+{
+namespace PhreeqcIOData
+{
+struct AqueousSolution;
+struct KineticReactant;
+struct EquilibriumReactant;
+
+struct ChemicalSystem
+{
+    ChemicalSystem(std::unique_ptr<AqueousSolution>&& aqueous_solution_,
+                   std::vector<KineticReactant>&& kinetic_reactants_,
+                   std::vector<EquilibriumReactant>&& equilibrium_reactants_)
+        : aqueous_solution(std::move(aqueous_solution_)),
+          kinetic_reactants(std::move(kinetic_reactants_)),
+          equilibrium_reactants(std::move(equilibrium_reactants_))
+    {
+    }
+
+    std::unique_ptr<AqueousSolution> aqueous_solution;
```

Comment:
This requires include of "AqueousSolution.h" because of incomplete type...
https://jenkins.opengeosys.org/blue/organizations/jenkins/ufz%2Fogs/detail/PR-2982/1/pipeline#step-354-log-236

Reply:
Fixed, but I don't understand why it can be passed in serial run and on my local...

Reply:
Different order of includes could be, or unity builds.

Reply:
Thanks for your explanation.